core/src/utils/overlays.ts

```diff
@@ -574,6 +579,11 @@ export const dismiss = async <OverlayDismissOptions>(
     if (role !== GESTURE) {
       await overlayAnimation(overlay, animationBuilder, overlay.el, opts);
     }
+
+    if (lastOverlay) {
+      document.body.classList.remove(BACKDROP_NO_SCROLL);
```

Comment:
Out of curiosity, are we able to remove the class when we do `setRootAriaHidden(false)`? The underlying content shouldn't be scrollable since the backdrop will still catch pointer events until it's been removed from the DOM.

Reply:
Confirmed that we can! Originally I thought about avoiding the overflow layout repaint until after the dismiss transition, but it doesn't seem to make an observable impact to do it prior.